app/src/main/java/edu/usf/sas/pal/muser/util/EventFirebaseIOUtils.java

```diff
@@ -0,0 +1,51 @@
+package edu.usf.sas.pal.muser.util;
+
+import android.util.Log;
+
+import com.google.firebase.firestore.DocumentReference;
+import com.google.firebase.firestore.FirebaseFirestore;
+
+import edu.usf.sas.pal.muser.constants.EventConstants;
+import edu.usf.sas.pal.muser.model.Event;
+
+public class EventFirebaseIOUtils {
```

Comment:
Per my other comment, let's change the name of this class to the more generic `FirebaseIOUtils`:

```suggestion
public class FirebaseIOUtils {
```